src/main/java/games/strategy/triplea/delegate/EditValidator.java

```diff
@@ -144,7 +140,7 @@ public static String validateRemoveUnits(final GameData data, final Territory te
      * if (!Match.allMatch(units, Matches.unitIsOwnedBy(player)))
      * return "Not all units have the same owner";
      */
-    if ((result = validateTerritoryBasic(data, territory, player)) != null) {
```

Comment:
Local variable `player` (declared above on line 137) is now unused. Leaving it in place to preserve any possible side effects caused by the expression `units.iterator().next().getOwner()`.